aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -66,87 +66,52 @@ Recommendations for global deployments to geographical data centers:
 
 :::moniker-end
 
-:::moniker range="< aspnetcore-8.0"
+For guidance on deployment and configuration for the Azure SignalR Service:
 
-Hosting a Blazor Server app on Azure App Service requires configuration for Application Request Routing (ARR) affinity and WebSockets. The App Service should also be appropriately globally distributed to reduce UI latency. Using the Azure SignalR Service when hosting on Azure App Service isn't required.
+* Only when the app is in production on Azure App Service, see <xref:signalr/publish-to-azure-web-app>.
+* When you want the app to adopt the Azure SignalR Service during development, see [Build Blazor Server chat app: Further topic: Enable Azure SignalR Service in local development](https://github.com/aspnet/AzureSignalR-samples/tree/main/samples/BlazorChat#further-topic-enable-azure-signalr-service-in-local-development).
 
-:::moniker-end
+We recommend using WebSockets for server-side Blazor apps deployed to Azure App Service. If [WebSockets](https://wikipedia.org/wiki/WebSocket) are disabled, Azure App Service simulates a real-time connection using HTTP Long Polling. HTTP Long Polling is noticeably slower than running with WebSockets enabled, which doesn't use polling to simulate a client-server connection.
+
+In the event that Long Polling must be used, you may need to configure the maximum poll interval (`MaxPollIntervalInSeconds`, default: 5 seconds, limit: 1-300 seconds), which defines the maximum poll interval allowed for Long Polling connections in the Azure SignalR Service if the service ever falls back from WebSockets to Long Polling. If the next poll request doesn't arrive within the maximum poll interval, Azure SignalR Service closes the client connection. The Azure SignalR Service also cleans up connections when the cached waiting-to-write buffer size is greater than 1 MB for improved service performance.
 
-1. Use the following guidance to configure the service:
+For more information, see:
 
-  * [Configure the app in Azure App Service](xref:signalr/publish-to-azure-web-app#configure-the-app-in-azure-app-service).
-  * [App Service Plan Limits](xref:signalr/publish-to-azure-web-app#app-service-plan-limits).
+* [Azure SignalR Service](/products/signalr-service/)
+* [What is Azure SignalR Service?](/azure/azure-signalr/signalr-overview)
+* <xref:signalr/scale#azure-signalr-service>
+* [Performance guide for Azure SignalR Service](/azure/azure-signalr/signalr-concept-performance#performance-factors)
+* <xref:signalr/publish-to-azure-web-app>
+* <xref:signalr/configuration>
+* [Build Blazor Server chat app](https://github.com/aspnet/AzureSignalR-samples/tree/main/samples/BlazorChat)
```

Comment:
I get a 404 for the first link in this list.

Some of the other links seem superfluous or irrelevant. How about this?:

```suggestion
* [What is Azure SignalR Service?](/azure/azure-signalr/signalr-overview)
* <xref:signalr/scale#azure-signalr-service>
* <xref:signalr/publish-to-azure-web-app>
```

Reply:
WIP ... 

That first link should go here ... https://azure.microsoft.com/en-us/products/signalr-service/ ... to the main MS page that advertises it. I clipped it for the learn domain ... ***BAD*** 🦖! 😄 

The perf link was just a holdover from ages ago.

The chat app link was added because thus far I'm relying on that cross-link to explain the use of the service, including the app configuration for it. One of the things that this PR is seeking to do is cross-link for the guidance in more stable locations instead of trying to keep up with it here.

Reply:
> The chat app link was added because thus far I'm relying on that cross-link to explain the use of the service, including the app configuration for it. One of the things that this PR is seeking to do is cross-link for the guidance in more stable locations instead of trying to keep up with it here.

I agree with this idea in principle, but that sample needs some love before I'm comfortable linking to it. The sample README says to install .NET Core 3.0 and VS 2019 😆.

Even if we eventually link to that sample, this page still needs to be our official guidance on how to use the Azure SignalR Service with Blazor, so that it's part of the official Blazor doc set (indexed for search, localized, etc.)

Reply:
Yes 😄 ... This is a WIP, and my pattern is not to cross-link when I don't like what I see (e.g., Azure Graph/AD groups & roles ... *I still like our stuff better!*). I have a feeling that I'll drop the cross-link for our main coverage tomorrow and just carry the guidance here. However, I should look around a little more for guidance similar to what that README has that I can cross-link. It's just that I think they are on the right track with how they summarized everything.